hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -162,9 +162,10 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     /**
      * @param readCount Number of items in the response.
```

Comment:
is this number needed? the size of the items list would be the same information, right?